applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc

```diff
@@ -45,7 +45,6 @@ include::modules/sbo-unbinding-workloads-from-a-backing-service.adoc[leveloffset
 [id="additional-resources_binding-workloads-sbo"]
 == Additional resources
 * xref:../../applications/connecting_applications_to_services/understanding-service-binding-operator.adoc#binding-a-workload-together-with-a-backing-service_understanding-service-binding-operator[Binding a workload together with a backing service].
-* xref:../../applications/connecting_applications_to_services/getting-started-with-service-binding.adoc#connecting-the-spring-petclinic-sample-application-to-the-postgresql-database-service[Connecting the Spring PetClinic sample application to the PostgreSQL database service].
+* xref:../../applications/connecting_applications_to_services/getting-started-with-service-binding#sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service_getting-started-with-service-binding[Connecting the Spring PetClinic sample application to the PostgreSQL database service].
```

Comment:
@mburke5678 Any particular reason why we are removing the `.adoc` from here? AFAIK, this is the xref format that is generally followed in OCP docs repo:

xref:../../dir/subdir/assembly.adoc#module-title_assembly-context-id (if you don't have subdir, you can leave only that part out)